### ISSUE TITLE
Some doubts about evaluation indicators (IS, FID)

### ISSUE THREAD

**Issue by dzl20221020** [prajwalsingh/EEGStyleGAN-ADA#29]
Thank you very much for your work. I am currently continuing your work and improving it. Then I hope to use your experimental results as the baseline of my work. I will try to evaluate the checkpoints you gave.
`python calc_metrics.py --metrics=is50k --network=out/00000-EEGImageCVPR40-cond-mirror-cifar-bgcfnc/network-snapshot-005322.pkl --data=../dataset/eeg_imagenet40_cvpr_2017_raw/test/* --mirror=1
`
![image](https://github.com/user-attachments/assets/8eda2533-7072-4d29-9a6b-6185f455ad1a)
When I used the IS evaluation index, I found that the results were almost the same as those given in the paper.But when I used FID for evaluation, I found that the difference was a bit big.
`python calc_metrics.py --metrics=fid50k_full --network=out/00000-EEGImageCVPR40-cond-mirror-cifar-bgcfnc/network-snapshot-005322.pkl --data=../dataset/eeg_imagenet40_cvpr_2017_raw/test/* --mirror=1
`
![image](https://github.com/user-attachments/assets/db32a40c-0814-4a4a-8e16-eecccfbddcb8)
Of course, I found that when using FID evaluation in my method, it is basically around 470. I feel very strange. I wonder if there is something wrong with me. The difference is so big. I hope you can help me. I will be very grateful.Moreover, there will be a small negative number when evaluating FID at checkpoints with insufficient training. However, I checked the information and said that it is caused by under-training and model instability.
I very much hope you can answer my questions. If I am lucky enough to publish a paper, I will thank and cite your work in the paper.

**Comment by prajwalsingh** [comment]
Hi @dzl20221020, thank you for reading our work and the acknowledgment. 

I tried rerunning the evaluation and got a score of around 210.368 FID (Quadro RTX 5000).

![eegstylegan](https://github.com/user-attachments/assets/7e6a9677-2f9a-43ce-bcf0-137e79731c7c)

This might be due to the randomness of the seed value. To confirm that, I checked StyleGAN-ADA GitHub, and following I found

![image](https://github.com/user-attachments/assets/b2caa4af-23a7-4068-9fb2-f20f3b66cda4)

Please feel free to report the values you are getting; it may also be due to the architecture difference or some libraries. You can also mention it as paper / self-run.

You can also enable FID computation after each set of epochs, which may slow down the training, but you can get the best FID possible in the long run. 

Please feel free to post any questions you have.

**Comment by dzl20221020** [comment]
Thank you for your answer. I tried the above test with RTX3090, and then I saw that you tried it with V100, so I tried it twice with A100 and found that the results were the same both times. I don’t know why it depends on the graphics card model. related.feels strange
![image](https://github.com/user-attachments/assets/8ee09c39-e582-4dc3-960a-856c8f26b3c2)
Thank you again for your answer and I look forward to seeing your better work
